scripts/Muon/GUI/ElementalAnalysis/PeriodicTable/periodic_table_presenter.py

```diff
@@ -0,0 +1,7 @@
+class PeriodicTablePresenter(object):
+    def __init__(self, view):
+        self.view = view
+
```

Comment:
we should connect to the view signals in here and have the slots/functions in here

Reply:
Do you mean the initial connects (i.e. view.table_clicked etc.) or the event registration? Thought it was better to expose the event registration from the widget class, but I could see why the initial connects could belong in the presenter.

Reply:
after reading some of the later code, I think I understand why you have done it. If we have an external widget for the user to load their own data it makes sense. Also we need to send some sort of signal (name and lines) to the plotting. Is that what you were thinking?

Reply:
Yeah exactly - makes it easier for inter-widget interaction.